### ISSUE TITLE
Incorrect time for email-fetched tickets

### ISSUE THREAD

**Issue by toxpal** [osTicket/osTicket#5786]
I think it's a big issue, but couldn't find any open/closed issue regarding this (please share the link to the original issue if it already exists and I missed it somehow).

All my tickets are created from email and work fine, except time displayed in tickets. Since I don't get hundreds of tickets a day, email cron is scheduled to run every 10 minutes. And this is where problem starts...

Let's say cron was launched 13.00 last time and 3 new emails were received during next 10 minutes. 1st email received at 13:01, 2nd - 13:04, 3rd: 13:08). When cron is launched, it incorrectly displays that all tickets were created at 13:10. I realize that cron was launched at 13:10, but shouldn't tickets display actual message time instead of cron launch time?

While +/-10 minutes don't change much, imagine someone runs cron once a day. In this situation, all emails received during last 24 hours will display exactly the same time, creating a big mess. For know, I could only think of one solution - run cron every single minute, but that's a waste of server resources in most of situations.

I realize it's possible to view actual email time when viewing email headers, but that doesn't help because all other places (tickets list, ticket view, etc.) display cron execution time only. If OSTicket needs to display cron execution time for some reason, maybe it would be possible to display email receive time in addition? Especially because every ticket has 2 fields:

Create Date
Last Message

And both of them display cron time instead of email receive time. Maybe we could "sacrifice" one field to display actual time of message?

**Comment by Mike-Benoit** [comment]
I brought this issue up years ago when we first migrated to osTicket and were initially importing historical tickets from years past. All the tickets had timestamps of today when they originally came in years ago. The osTicket Dev's unfortunately don't see this as a problem or at least not a priority. :(

**Comment by toxpal** [comment]
Yes, I forgot to mention that this issue is even worse when importing tickets.

**Comment by theking81** [comment]
Yes when for example I stop osticket from receiving emails (from dashboard settings) and and enabling it after 2-3 days all imported emails will have the current timestamp and not the date the sender send it.
It would be great to have an option to choose how we prefer to get them listed.
